Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Login/LogInViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  ViewController.swift
+//  Internship2022TeamPatri
+//
+//  Created by Szabolcs Orban on 25.07.2022.
+//
+
+import UIKit
+import SnapKit
+
+class LogInViewController: UIViewController {
+    
+    @IBOutlet weak private var passwordLabel: UILabel!
+    @IBOutlet weak private var emailLabel: UILabel!
+    @IBOutlet weak private var logInLabel: UILabel!
+    @IBOutlet weak private var emailTextField: UITextField!
+    @IBOutlet weak private var passwordTextField: UITextField!
+    @IBOutlet weak private var loginButton: UIButton!
+    @IBOutlet weak private var notRegisteredLabel: UILabel!
+    @IBOutlet weak private var signupButton: UIButton!
+    @IBOutlet weak private var forgotPasswordButton: UIButton!
```

Comment:
Please be consistent when naming your labels/variables/etc (logInLabel, loginButton). Please check the other variables as well